docs/tables/mongodbatlas_container.md

```diff
@@ -0,0 +1,23 @@
+# Table: mongodbatlas_container
+
+Containers in a `mongodbatlas` project allows for cloud provider backed virtual private networking - dubbed as `container network peering` in MongoDB Atlas
```

Comment:
```suggestion
Containers in a MongoDB Atlas project allows for cloud provider backed virtual private networking - dubbed as `container network peering` in MongoDB Atlas.
```

All descriptions should be full sentences and end in periods as per https://steampipe.io/docs/develop/table-docs-standards#style-conventions